src/popup/module/EmojiPicker.js

```diff
@@ -5,6 +5,8 @@
  */
 
 import * as EmojiSelect from "./EmojiSelect.js";
+import data from "/node_modules/@emoji-mart/data";
```

Comment:
The problem in the official guide was that this is not an ECMAstandard apparently. This here should load a combined JSON, but dpoes not work in Firefox like this.

Reply:
Yeah, one cannot yet import JSON files as a module (see [here](https://github.com/tc39/proposal-json-modules)). I believe the solution would be to remove this line and then use a hack like this below:
```js
const emojiPicker = new EmojiMart.Picker({ ...initProperties, data: async () => {
    const response = await fetch(browser.runtime.getURL("/node_modules/@emoji-mart/data/sets/14/native.json"));
    return response.json();
}});
```